src/oAuth2.js

```diff
@@ -113,7 +113,8 @@ export class OAuth2 {
 
     return this.http.fetch(exchangeForTokenUrl, {
       method: 'post',
-      body: json(data),
+      headers: { 'Content-Type': 'application/json; charset=utf-8' },
+      body: JSON.stringify(data),
```

Comment:
just out of curiosity, does `body: json(data)` still work? As in, is it only required that we add the headers here?